api/openapi-spec/conventions-server.yaml

```diff
@@ -0,0 +1,153 @@
+        500:
+          description: Internal server error
+          content:
+            "application/json":
+              schema:
+                $ref: "#/components/schemas/ErrorResponse"
+components:
+  schemas:
+    PodConventionContext:
+      description: a wrapper for the PodConventionContextSpec and the PodConventionContextStatus.
+      type: object
+      properties:
+        apiVersion: 
+          type: string
+        kind: 
+          type: string
+        metadata:
+          type: object 
+          properties:
+            name: 
+              type: string
+        spec:
+          $ref: "#/components/schemas/PodConventionContextSpec"
+        status:
+          $ref: "#/components/schemas/PodConventionContextStatus"
+    PodConventionContextSpec:
+      type: object
+      description: a wrapper of the PodTemplateSpec and the ImageConfig provided in the request body of the server.
+      properties:
+        template:
+          $ref: "#/components/schemas/PodTemplateSpec"
+        imageConfig:
+          type: array 
+          description: |
+            an array of imageConfig objects with each image configuration object holding the name of the image, the BOM, and the OCI image
+            configuration with image metadata from the repository. Each of the image config array entries have a 1:1 mapping to 
+            images referenced in the PodTemplateSpec. 
+          items:
+            $ref: "#/components/schemas/ImageConfig"
+    PodTemplateSpec:
+      type: object 
+      properties:
+        spec: 
+          type: object
+          description: defines the PodTemplateSpec to be enriched by conventions.
+        metadata:
+          type: object
+          properties:
+            annotations:
+              type: object
+              additionalProperties:
+                type: string
+            labels:
+              type: object
+              additionalProperties:
+                type: string
+    ImageConfig:
+      type: object 
+      properties: 
+        image: 
+          type: string 
+          description: a string reference to the image name and tag or associated digest.
+          example: "example.com/repository/nginx:alpine"
+        BOMs:
+          type: array
+          description: | 
+            an array of Bills of Materials (BOMs) describing the software components and their dependencies and may be zero or more per image.
+          items:
+            $ref: "#/components/schemas/BOM"
+          example: | 
+            "boms": [{
+                "name": "bom-name",
+                "raw": "a byte array"
+            }],
+        config: 
+          type: object 
+          description: a ggcrv1 config file object.
+    BOM: 
+      type: object 
+      properties:
+        name: 
+          description: bom-name
+          type: string 
+        raw: 
+          description: a byte array with the encoded content of the BOM.
+          type: string
```

Comment:
I think it would be useful if the encoding format of the byte array was described here.  Is it base64-encoded?  Is it hexadecimal?